mcdc/loop.py

```diff
@@ -41,11 +47,11 @@ def loop_main(mcdc):
                 kernel.tally_closeout_history(mcdc)
 
             # Print progress
-            with objmode():
-                print_progress_eigenvalue(mcdc)
+            # with objmode():
+            # print_progress_eigenvalue(mcdc)
 
             # Manage particle banks
-            kernel.manage_particle_banks(mcdc)
+            kernel.manage_particle_banks(cycle_seed, mcdc)
```

Comment:
May ned to carefully treat this one too.

Reply:
Each argument now gets its own, unique hash. Convert the seed hex to ASCII for a fun surprise.